tiny-types/src/test/scala/io/renku/tinytypes/TinyTypeSpec.scala

```diff
@@ -29,12 +29,17 @@ import org.scalatest.matchers.should
 import org.scalatest.wordspec.AnyWordSpec
 import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
 
+object TinyTypeData {
+  val arbitraryValues: List[Any] =
```

Comment:
I know it's more performant to keep the values in a separate object but what do you think if we'd put the `arbitraryValues` val as a private member somewhere in the test class?

Reply:
I have no strong opinion here, can do that as well. It would be duplicated in three test classes I think, which is not problematic, just saying :)